src/wifi_manager.c

```diff
@@ -53,14 +53,19 @@ Contains the freeRTOS task and all necessary support
 #include "lwip/err.h"
 #include "lwip/netdb.h"
 #include "lwip/ip4_addr.h"
+#include "sdkconfig.h"
 
 
 #include "json.h"
 #include "dns_server.h"
 #include "nvs_sync.h"
 #include "wifi_manager.h"
 
-
```

Comment:
I'm not too sure about this. It adds nothing useful. The build system will always define CONFIG_WIFI_MANAGER_DNS_SERVER_ENABLE

Reply:
You should define the build time variables as per wifi_manager.h which is best practice. No need to define a new global byte.

Reply:
I'll have to validate, from what I remember when set to 'n' `CONFIG_WIFI_MANAGER_DNS_SERVER_ENABLE' was simply undefined. That why I check if it's defined or not. 

You're right though, this should be in `wifi_manager.h`.